remotes/docker/fetcher.go

```diff
@@ -18,13 +18,16 @@ package docker
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"path"
 	"strings"
 
+	"github.com/docker/distribution/registry/api/errcode"
+
```

Comment:
remove this empty line ?

Reply:
That new line is added by goimport which is run as part of the CI tests so believe it would cause the travis build to fail if I remove it

Reply:
if you add it after `containerd/containerd/log` you should be able to verify with `gofmt` that that would allow all external imports to be in the same section (which is pretty normal) rather than having it stand by itself. `goimport` isn't always as smart about combining these sections.

Reply:
Sorry I was looking at the _test file when I first read this. There is an extra space as you say. Will fix